src/Core/SettingsChangesHistory.h

```diff
@@ -111,6 +111,7 @@ static const std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges
               {"output_format_csv_serialize_tuple_into_separate_columns", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_deserialize_separate_columns_into_tuple", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_try_infer_strings_from_quoted_tuples", true, true, "A new way of how interpret tuples in CSV format was added."},
+              {"backup_restore_s3_retry_attempts", 0, 1000, "A new setting."},
```

Comment:
24.6 is released. This will need to be changed to 24.7

Reply:
Do not understand what you mean.

Reply:
Oh. I see it is rotten already because it was not merged on time.

Reply:
I already curious is it even possible to merge something without overriding CI block.